8Knot/assets/landing_page.css

```diff
@@ -191,253 +191,246 @@
-    font-size: 16px !important;
-    font-weight: 400 !important;
-    line-height: 1.6 !important;
-    margin-bottom: var(--spacing-sm) !important;
+.landing-page .feature-body p {
+    color: var(--color-text-secondary);
+    font-size: var(--landing-font-sm);
+    font-weight: 400;
+    line-height: 1.6;
+    margin-bottom: var(--spacing-sm);
 }
 
-.image-container {
-    flex: 1 !important;
-    text-align: center !important;
+.landing-page .image-container {
+    flex: 1;
+    text-align: center;
 }
 
-.feature-image {
-    max-width: 100% !important;
-    height: auto !important;
-    border-radius: var(--border-radius) !important;
-    border: 1px solid var(--color-border) !important;
+.landing-page .feature-image {
+    max-width: 100%;
+    height: auto;
+    border-radius: var(--border-radius);
+    border: 1px solid var(--color-border);
 }
 
-.image-caption {
-    color: var(--text-muted) !important;
-    font-size: 12px !important;
-    margin-top: var(--spacing-xs) !important;
-    font-style: italic !important;
+.landing-page .image-caption {
+    color: var(--color-text-muted);
+    font-size: 12px;
+    margin-top: var(--spacing-xs);
+    font-style: italic;
 }
 
 
 /* Side Navigation in Card */
 
-.tab-content-container {
-    background: var(--color-topbar-bg) !important;
-    padding: var(--spacing-lg) !important;
-    border-radius: 0 0 var(--border-radius-lg) var(--border-radius-lg) !important;
+.landing-page .tab-content-container {
+    background: var(--color-topbar-bg);
+    padding: var(--spacing-lg);
+    border-radius: 0 0 var(--border-radius-lg) var(--border-radius-lg);
 }
 
-.side-nav {
-    display: none !important;
+.landing-page .side-nav {
+    display: none;
     /* Hide side nav for cleaner look */
 }
 
-.tab-content-main {
-    width: 100% !important;
+.landing-page .tab-content-main {
+    width: 100%;
 }
 
 
 /* Responsive Design for Dark Theme Tabs */
 
 @media (max-width: 768px) {
-    .tabs-header {
-        padding: 0px var(--spacing-sm) !important;
-    }
-    .tabs-header .nav-tabs .nav-item:nth-child(1) .nav-link,
-    .tabs-header .nav-tabs .nav-item:nth-child(2) .nav-link,
-    .tabs-header .nav-tabs .nav-item:nth-child(3) .nav-link,
-    .tabs-header .nav-tabs .nav-item:nth-child(4) .nav-link,
-    .tabs-header .nav-tabs .nav-item:nth-child(5) .nav-link {
-        width: auto !important;
-        font-size: 14px !important;
-        padding: 6px 12px !important;
-        min-width: 120px !important;
+    .landing-page .tabs-header {
+        padding: 0px var(--spacing-sm);
+    }
+    .landing-page .tabs-header .nav-tabs .nav-item:nth-child(1) .nav-link,
+    .landing-page .tabs-header .nav-tabs .nav-item:nth-child(2) .nav-link,
+    .landing-page .tabs-header .nav-tabs .nav-item:nth-child(3) .nav-link,
+    .landing-page .tabs-header .nav-tabs .nav-item:nth-child(4) .nav-link,
+    .landing-page .tabs-header .nav-tabs .nav-item:nth-child(5) .nav-link {
```

Comment:
note to self, im pretty sure tabs-header is only used on the landing page, so this is probably okay, but if we wanted to reuse it elsewhere it may be a problem